code/modules/clothing/chameleon.dm

```diff
@@ -0,0 +1,252 @@
+	desc = "A slim armored vest that protects against most types of damage."
+	icon_state = "armor"
+	item_state = "armor"
+	blood_overlay_type = "armor"
+	origin_tech = "syndicate=3"
+	burn_state = FIRE_PROOF
+	armor = list(melee = 10, bullet = 10, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/suit/chameleon/New()
+	..()
+	var/datum/action/item_action/chameleon/change/chameleon_action = new
+	chameleon_action.target = src
+	chameleon_action.chameleon_type = /obj/item/clothing/suit
+	chameleon_action.chameleon_name = "Suit"
+	chameleon_action.initialize_disguises()
+	action = chameleon_action
+
+/obj/item/clothing/glasses/chameleon
+	name = "Optical Meson Scanner"
+	desc = "Used by engineering and mining staff to see basic structural and terrain layouts through walls, regardless of lighting condition."
+	icon_state = "meson"
+	item_state = "meson"
+
+	origin_tech = "syndicate=3"
+	burn_state = FIRE_PROOF
+	armor = list(melee = 10, bullet = 10, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/glasses/chameleon/New()
+	..()
+	var/datum/action/item_action/chameleon/change/chameleon_action = new
+	chameleon_action.target = src
+	chameleon_action.chameleon_type = /obj/item/clothing/glasses
+	chameleon_action.chameleon_name = "Glasses"
+	chameleon_action.initialize_disguises()
+	action = chameleon_action
+
+/obj/item/clothing/gloves/chameleon
+	desc = "These gloves will protect the wearer from electric shock."
+	name = "insulated gloves"
+	icon_state = "yellow"
+	item_state = "ygloves"
+
+	burn_state = FIRE_PROOF
+	armor = list(melee = 10, bullet = 10, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/gloves/chameleon/New()
+	..()
+	var/datum/action/item_action/chameleon/change/chameleon_action = new
+	chameleon_action.target = src
+	chameleon_action.chameleon_type = /obj/item/clothing/gloves
+	chameleon_action.chameleon_name = "Gloves"
+	chameleon_action.initialize_disguises()
+	action = chameleon_action
+
+/obj/item/clothing/head/chameleon
+	name = "grey cap"
+	desc = "It's a baseball hat in a tasteful grey colour."
+	icon_state = "greysoft"
+	item_color = "grey"
+
+	burn_state = FIRE_PROOF
+	armor = list(melee = 5, bullet = 5, laser = 5, energy = 0, bomb = 0, bio = 0, rad = 0)
+
+/obj/item/clothing/head/chameleon/New()
+	..()
+	var/datum/action/item_action/chameleon/change/chameleon_action = new
+	chameleon_action.target = src
+	chameleon_action.chameleon_type = /obj/item/clothing/head
+	chameleon_action.chameleon_name = "Hat"
+	chameleon_action.initialize_disguises()
+	action = chameleon_action
+
+/obj/item/clothing/mask/chameleon
+	name = "gas mask"
+	desc = "A face-covering mask that can be connected to an air supply. While good for concealing your identity, it isn't good for blocking gas flow." //More accurate
+	icon_state = "gas_alt"
+	item_state = "gas_alt"
+	burn_state = FIRE_PROOF
+	armor = list(melee = 5, bullet = 5, laser = 5, energy = 0, bomb = 0, bio = 0, rad = 0)
```

Comment:
Why? Just keep it at regular gas mask levels. Chameleon stuff isn't supposed to be armored.